zstash/globus.py

```diff
@@ -22,9 +23,11 @@
 regex_endpoint_map = {
     r"theta.*\.alcf\.anl\.gov": "08925f04-569f-11e7-bef8-22000b9a448b",
     r"blueslogin.*\.lcrc\.anl\.gov": "61f9954c-a4fa-11ea-8f07-0a21f750d19b",
+    r"chrlogin.*\.lcrc\.anl\.gov": "61f9954c-a4fa-11ea-8f07-0a21f750d19b",
     r"b\d+\.lcrc\.anl\.gov": "61f9954c-a4fa-11ea-8f07-0a21f750d19b",
     r"chr.*\.lcrc\.anl\.gov": "61f9954c-a4fa-11ea-8f07-0a21f750d19b",
     r"cori.*\.nersc\.gov": "9d6d99eb-6d04-11e5-ba46-22000b92c6ec",
+    r"compy.*\.pnl\.gov": "68fbd2fa-83d7-11e9-8e63-029d279f7e24",
     r"perlmutter.*\.nersc\.gov": "6bdc7956-fc0f-4ad2-989c-7aa5ee643a79",  # If this doesn't work, use cori
```

Comment:
You make a good point, including the compute nodes for Compy. I think Perlmutter compute nodes also don't fit this pattern.

I usually run `zstash` from a login node but I suppose some may find it useful to run a large Globus transfer from a compute node.

Reply:
@mahf708 The issue I discovered with using compute nodes is that sometimes the Globus transfers take longer than the max wall clock time (12 hours according to https://docs.nersc.gov/jobs/policy/). That is, I think we'd need to set up some sort of restart file system (as E3SM itself does) and use `zstash --update`.